### PR TITLE
Add detailed information in moreinfo field

### PR DESCRIPTION
**Pull request by flands**
Add detailed information in moreinfo. Add basic auth logging in test endpoint. Updated tests.

### CODE REVIEW

File: tests/init_test.py [hunk 1]
```diff
@@ -224,6 +210,6 @@ def test_wrong_method():
 
 def test_test():
     rsp = conftest.client.post('/endpoint/test')
-    assert rsp.status == '200 OK'
+    assert rsp.status == '500 INTERNAL SERVER ERROR'
```
Comment (alanjcastonguay) [inline tests/init_test.py]: Why is this changed?

File: loginsightwebhookdemo/__init__.py [hunk 2]
```diff
@@ -251,8 +258,17 @@ def _introduction():
...
 def test(ALERTID=None):
-    """Log the request and respond with success. Don't send the payload anywhere."""
+    """Log the auth header, request, and parsed moreinfo field. Respond with success. Don't send the payload anywhere."""
+    try:
+        logging.info(request.headers['Authorization'])
+    except KeyError:
+        pass
     logging.info(request.get_data())
+    try:
+        a = parse(request)
```
Comment (alanjcastonguay) [inline loginsightwebhookdemo/__init__.py]: This wasn't in your `try` block before, and probably shouldn't be -- if the body can't be parsed then raising an exception is appropriate; falling through to `return "OK"` isn't.

Comment (flands) [reply loginsightwebhookdemo/__init__.py]: The only reason to parse is for print on line 269. If either fail, it is OK (parse is tested separately already). What is the concern with including it in the try?

Comment (alanjcastonguay) [reply loginsightwebhookdemo/__init__.py]: Including it hides other (possibly future) problems.

Comment (flands) [reply loginsightwebhookdemo/__init__.py]: OK, will update momentarily.
